course-mw/enrolment-actor/src/main/scala/org/sunbird/enrolments/ContentConsumptionActor.scala

```diff
@@ -27,6 +27,20 @@ class ContentConsumptionActor @Inject() extends BaseEnrolmentActor {
     private val consumptionDBInfo = Util.dbInfoMap.get(JsonKey.LEARNER_CONTENT_DB)
     private val assessmentAggregatorDBInfo = Util.dbInfoMap.get(JsonKey.ASSESSMENT_AGGREGATOR_DB)
     val dateFormatter = ProjectUtil.getDateFormatter
+    val defaultFields = new java.util.ArrayList[String](){{
+        add("contentid")
+        add("userid")
+        add("batchid")
+        add("courseid")
+        add("completedcount")
+        add("completionpercentage")
+        add("lastcompletedtime")
+        add("progress")
+        add("status")
+        add("viewcount")
+    }}
```

Comment:
@pritha-tarento - Any reason why default fields have been added? and if they are needed, can the default fields be taken up from a configuration file?